src/Tools/Pagination/LimitSubqueryOutputWalker.php

```diff
@@ -149,7 +149,9 @@ private function rebuildOrderByForRowNumber(SelectStatement $AST): void
         $selectAliasToExpressionMap = [];
         // Get any aliases that are available for select expressions.
         foreach ($AST->selectClause->selectExpressions as $selectExpression) {
-            $selectAliasToExpressionMap[$selectExpression->fieldIdentificationVariable] = $selectExpression->expression;
+            if (is_string($selectExpression->fieldIdentificationVariable)) {
```

Comment:
Let's check for null instead.


```suggestion
            if ($selectExpression->fieldIdentificationVariable !== null) {
```

Reply:
done